incubator/cloudflare-datadog/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Can you add an `icon` and `appVersion`?

Also, please use Github usernames instead of real names.